### PR TITLE
WDP240901-16

### PR DESCRIPTION
**Pull request by dave8git**
https://projects.kodilla.com/browse/WDP240901-16

### CODE REVIEW

File: src/components/features/FeatureBoxes/FeatureBoxes.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import React from 'react';
```
Comment (patrykbn) [inline src/components/features/FeatureBoxes/FeatureBoxes.js:1]: white space after ;

File: src/components/features/FeatureBoxes/FeatureBoxes.js [hunk 2]
```diff
@@ -9,32 +9,32 @@ import {
 } from '@fortawesome/free-solid-svg-icons';
 
 import styles from './FeatureBoxes.module.scss';
-import FeatureBox from '../../common/FeatureBox/FeatureBox';
+import FeatureBox from '../../common/FeatureBox/FeatureBox';
```
Comment (patrykbn) [inline src/components/features/FeatureBoxes/FeatureBoxes.js:12]: white-space after ;

File: src/components/features/FeatureBoxes/FeatureBoxes.module.scss [hunk 3]
```diff
@@ -3,3 +3,6 @@
 .root {
   padding: 5rem 0;
 }
+
```
Comment (patrykbn) [inline src/components/features/FeatureBoxes/FeatureBoxes.module.scss:6]: delete empty lines

**Review by patrykbn — CHANGES_REQUESTED** [review]
Just a few white-spaces need to be deleted to clean up code

**Review by patrykbn — DISMISSED** [review]

File: src/components/features/FeatureBoxes/FeatureBoxes.js (hunk 1 above)
Comment (matwjablonski) [reply src/components/features/FeatureBoxes/FeatureBoxes.js:1]: you didn't remove that white space

File: src/components/features/FeatureBoxes/FeatureBoxes.js (hunk 2 above)
Comment (matwjablonski) [reply src/components/features/FeatureBoxes/FeatureBoxes.js:10]: same

File: src/components/common/FeatureBox/FeatureBox.module.scss [hunk 4]
```diff
@@ -7,6 +7,9 @@
   border: 1px solid $lighterGray;
   text-align: center;
   margin-top: 40px;
+  @media (max-width: 490px) {
```
Comment (matwjablonski) [inline src/components/common/FeatureBox/FeatureBox.module.scss:10]: before @media query add one empty line

**Review by matwjablonski — CHANGES_REQUESTED** [review]

File: src/components/features/FeatureBoxes/FeatureBoxes.js (hunk 1 above)
Comment (dave8git) [reply src/components/features/FeatureBoxes/FeatureBoxes.js]: white space removed.

File: src/components/features/FeatureBoxes/FeatureBoxes.js (hunk 2 above)
Comment (dave8git) [reply src/components/features/FeatureBoxes/FeatureBoxes.js]: white space removed.

**Review by patrykbn — APPROVED** [review]

**Review by matwjablonski — APPROVED** [review]

**Pull request merged**